test cases/diasdk/1 simple/meson.build

```diff
@@ -0,0 +1,8 @@
+project('diatest', 'cpp')
```

Comment:
I think you should rather add the tests to `test cases/windows` instead of creating a new test category

Reply:
Yeah, I think Windows would be appropriate. The other option would be `frameworks`